isv.md

```diff
@@ -11,6 +11,7 @@ Additionally:
 
 | Category | ISV | Product | Resources |
 | --- | --- | --- | --- |
+| Analytics | [Confluent](https://partners.amazonaws.com/partners/001E000001OMadFIAT/Confluent,%20Inc.) | [Confluent Platform](https://www.confluent.io/product/confluent-platform/) | Announcement: https://www.confluent.io/blog/introducing-confluent-platform-7-6/#arm64-support<br>Documentation: [https://docs.databricks.com/clusters/graviton.html](https://docs.confluent.io/platform/current/installation/system-requirements.html) |
```

Comment:
[https://docs.databricks.com/clusters/graviton.html](https://docs.confluent.io/platform/current/installation/system-requirements.html)

Looks like a typo, it lists in the markdown as a link to databricks, but goes to the confluent documentation.  Please update.

Reply:
Fixed the typo. Thanks!